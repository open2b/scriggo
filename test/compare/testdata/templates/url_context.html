{# render #}

<a href="">
<a href="abc">
<a href="æœ¬">
<a href="{{ `b` }}">
<a href="{{ `/` }}">
<a href="{{ `http://www.site.com/` }}">
<a href="{{ ` http://www.site.com/ ` }}">
<a href="http://s/{{ `aÃ æœ¬` }}/">
<a href="{{ `a` }}{{ `æœ¬` }}">
<a href="{{ `a` }}?b={{ ` ` }}">
<a href="{{ `a` }}?b={{ `=` }}">
<a href="{{ `=` }}?b={{ `=` }}">
<a href="{{ `p?` }}?b={{ `=` }}">
{# <a href="{{ `p?&` }}?b={{ `=` }}"> #}
<a href="{{ `p?q` }}?b={{ `=` }}">
<a href="{{ `?` }}?b={{ `=` }}">
<a href="{{ `/a/b/c` }}?b={{ `=` }}&c={{ `6` }}">
<a href="{{ `/a/b/c` }}?b={{ `=` }}&amp;c={{ `6` }}">
<a href="{{ `` }}?{{ `` }}">
<a href="?{{ `b` }}">
<a href="?{{ `=` }}">
<a href="#">
<a href="#{{ `=` }}">
<a href="{{ `=` }}#{{ `=` }}">
<a href="{{ `=` }}?{{ `=` }}#{{ `=` }}">
<a href="{{ `=` }}?b=6#{{ `=` }}">
<a href="{{ `,` }}?{{ `,` }}">
<img srcset="{{ `large.jpg` }} 1024w, {{ `medium.jpg` }} 640w,{{ `small.jpg` }} 320w">
<img srcset="{{ `large.jpg?s=1024` }} 1024w, {{ `medium.jpg` }} 640w">
<img srcset="{{ `large.jpg?s=1024` }} 1024w, {{ `medium=.jpg` }} 640w">
<a href="{% if true %}{{ `=` }}{% else %}?{{ `=` }}{% end %}">
<a href="{% if false %}{{ `=` }}{% else %}?{{ `=` }}{% end %}">
<input {{ `disabled` }}>
<a href={{ `b` }}>
<a href={{ ` b ` }}>
<a href= {{ ` b `}} >
<a href= {{ "\t\n\r\x0C b=`" }} >
<a href="{{ (map[interface{}]interface{}{})[`a`] }}">