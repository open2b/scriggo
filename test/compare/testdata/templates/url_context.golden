

<a href="">
<a href="abc">
<a href="æœ¬">
<a href="b">
<a href="/">
<a href="http://www.site.com/">
<a href=" http://www.site.com/ ">
<a href="http://s/a%c3%a0%e6%9c%ac/">
<a href="a%e6%9c%ac">
<a href="a?b=%20">
<a href="a?b=%3d">
<a href="=?b=%3d">
<a href="p?b=%3d">
// <a href="p?&amp;b=%3d">
<a href="p?q&amp;b=%3d">
<a href="?b=%3d">
<a href="/a/b/c?b=%3d&c=6">
<a href="/a/b/c?b=%3d&amp;c=6">
<a href="?">
<a href="?b">
<a href="?%3d">
<a href="#">
<a href="#%3d">
<a href="=#%3d">
<a href="=?%3d#%3d">
<a href="=?b=6#%3d">
<a href=",?%2c">
<img srcset="large.jpg 1024w, medium.jpg 640w,small.jpg 320w">
<img srcset="large.jpg?s=1024 1024w, medium.jpg 640w">
<img srcset="large.jpg?s=1024 1024w, medium=.jpg 640w">
<a href="=">
<a href="?%3d">
<input disabled>
<a href=b>
<a href=&#32;b&#32;>
<a href= &#32;b&#32; >
<a href= %09%0a%0d%0c&#32;b=%60 >
<a href="">